content/en/real_user_monitoring/session_replay/developer_tools.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Session Replay Developer Tools
+kind: documentation
+description: Describes the developer tools available in Session Replay
+aliases:
+further_reading:
+    - link: '/real_user_monitoring/session_replay'
+      tag: Documentation
+      text: Session Replay
+---
+
+<div class="alert alert-warning">Developer Tools is in public beta.</div>  
+
+## Overview
+
+Session Replay's Developer Tools are built-in debugging tools that can help you troubleshoot issues in your applications. There is no configuration needed for Dev Tools. 
+
+## Dev Tools
+
+To access Dev Tools, either click the **Jump to Replay** button to the left of a session in the **Sessions** tab or click on a session and click **Replay Session** on the top right corner in the [RUM Explorer][1]. 
+
+{{< img src="real_user_monitoring/session_replay/dev-tools-beta.png" alt="Dev Tools button" style="width:80%;">}}
```

Comment:
This image shows up weird in the preview: https://a.cl.ly/9ZuDdRBl

Reply:
```suggestion
{{< img src="real_user_monitoring/session_replay/dev_tools/dev-tools-beta.png" alt="Dev Tools button" style="width:80%;">}}
```